STYLEGUIDE.md

```diff
@@ -1,39 +1,311 @@
+  await user.update({
+    firstName: "Twilight",
+    lastName: "Sparkle",
+  });
+};
+```
+
+This assumes the learner is familiar with the cartoon _My Little Pony: Friendship is Magic,_ which is irrelevant.
+
+### 3.3 Use "they/them/theirs" pronouns
+
+For many years it was common to write with "he/him/his" pronouns in English technical documentation. But pronouns are difficult to translate into some languages. Gendering learners also sends the message that some genders are "default"—and this might not be the learner's gender. We use the neutral "they/them/theirs" pronouns.
+
+#### Do
+
+> When the user logs out using the button, they will be taken to the login page.
+
+#### Don't
+
+> When he logs out using the button, he will be taken to the login page.
+
+### 3.4 Avoid "click"
+
+"Click" is an outdated term that assumes the learner is using a mouse. But learners may be navigating by touchscreen, keyboard, or assistive technology. Often there are better words than "click"—like "select" and "open".
+
+#### Do
+
+> Open the **Settings tab.**
+
+> Select the **Google** social connection.
+
+#### Don't
+
+> Click the **Settings tab.** 
+
+> Click the **Google** social connection.
+
+## 4. Write for code
+
+### 4.1 Use monospace fonts for code, commands, file names, copy-ables, and URLs; use bold for menus, and other UI copy.
+
+#### Do
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to **User & Authentication > Social Connections** and open the **Settings tab.**
+
+> In your browser, open [`http://localhost:3000/`](http://localhost:3000/).
+
+#### Don't
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to `User & Authentication > Social Connections` and open the "Settings tab."
+
+> In your browser, open [`http://localhost:3000/`](http://localhost:3000/).
+
+### 4.2 Use carets to help users navigate menus; 
+
+Use carets to nest operations of the same type like menu navigation.
+
+#### Do
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to **User & Authentication > Social Connections** and open the **Settings tab.**
+
+#### Don't
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to **User & Authentication**, click on **Social Connections**, and open the **Settings tab.**
+
+
+### 4.3 Write compliant JSON: no single quotes, no comments
+
+Conform to the [JSON standard](https://www.json.org/json-en.html):
+
+* Key names are enclosed in double quotes (`"`)
+* Values are between double quotes (`"`), not single quotes (`'`)
+* No comments, neither `//` nor `/* */`
+
+#### Do
+
+> This is the user's primary email address in the User object
+
+```json
+{
+  "email": "{{user.primary_email_address}}"
+}
+```
+
+#### Don't
+
+```json
+{
+	// this is the user's primary email address in the User object
+  email: '{{user.primary_email_address}}'
+}
+```
+ 
+### 4.4 Use 2-space indents for code blocks
+
+We use 2-space indents to conserve horizontal space in code blocks.
```

Comment:
Can we enforce this with automation or tooling?

Reply:
I would love to have or help setup a prettier that can be used to format code before adding it to dpcs.

Reply:
That's the plan, Roy! As soon as this PR is merged/approved by all, we can start linting for as many of these as possible.

Reply:
Nice! I would also propose making them available in the repo if possible, so contributors can use (if that is not part of the plans already)